routes/outbound-lims.xml

```diff
@@ -8,21 +8,27 @@
                 <simple>${body.tableName} == 'test_order'</simple>
                 <setProperty name="lab-order-uuid">
                     <simple>${body.identifier}</simple>
-                </setProperty>
+                </setProperty>                
                 <setBody>
                     <simple>{{openmrs.username}}:{{openmrs.password}}</simple>
                 </setBody>
                 <marshal>
                     <base64/>
                 </marshal>
                 <convertBodyTo type="java.lang.String"/>
-                <setHeader name="Authorization">
+                
+                <setProperty name="basic-openmrs-authentication">
                     <simple>Basic ${body}</simple>
+                </setProperty>
+
+                <setHeader name="Authorization">
+                    <simple>${exchangeProperty.basic-openmrs-authentication}</simple>
                 </setHeader>
+
                 <setHeader name="CamelHttpMethod">
                     <constant>GET</constant>
                 </setHeader>
-
+                
                 <!--retreiving service request from openmrs fhir-->
 
                 <toD uri="{{openmrs.baseUrl}}/ws/fhir2/R4/ServiceRequest/${exchangeProperty.lab-order-uuid}"/>
```

Comment:
Sounds like we should refactor this `toD` URI to not pass the `${exchangeProperty.lab-order-uuid}` to the URI, as it would dynamically create a new endpoint for each new UUID.
Could you rather use:
```
 <setHeader name="CamelHttpQuery">
    <constant>${exchangeProperty.lab-order-uuid}</constant>
</setHeader>
<toD uri="{{openmrs.baseUrl}}/ws/fhir2/R4/ServiceRequest/" />
```

and actually, we wouldn't need the `toD` at all I believe, `to` should suffice.

Reply:
>as it would dynamically create a new endpoint for each new UUID

I'm not sure I understand the draw back clearly here, could you please elaborate this a little bit more?